cmd/start/flags.go

```diff
@@ -2,34 +2,33 @@ package start
 
 import (
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 	"github.com/spf13/viper"
 
 	"github.com/zitadel/zitadel/cmd/key"
 	"github.com/zitadel/zitadel/cmd/tls"
 )
 
-var tlsMode *string
+var (
+	tlsMode *string
 
-func startFlags(cmd *cobra.Command) {
-	bindUint16Flag(cmd, "port", "port to run ZITADEL on")
-	bindStringFlag(cmd, "externalDomain", "domain ZITADEL will be exposed on")
-	bindStringFlag(cmd, "externalPort", "port ZITADEL will be exposed on")
+	portFlag           = "port"
+	externalDomainFlag = "externalDomain"
+	externalPortFlag   = "externalPort"
```

Comment:
I don't see much functional reason why these are moved into variables. It is cleaner, to read yes. But then I would suggest to make them contants. These values are never supposed to change.